src/main/java/org/broadinstitute/hellbender/tools/pon/allelic/AllelicPanelOfNormals.java

```diff
@@ -317,6 +320,47 @@ private static HyperparameterValues parseGlobalHyperparameterValues(final File i
         }
     }
 
+    /**
+     * Find MLE hyperparameter values from counts in panel of normals.  See analogous code in {@link AlleleFractionInitializer}.
+     */
+    private HyperparameterValues calculateMLEGlobalHyperparameterValues(final AllelicCountCollection counts) {
+        double meanBias = AlleleFractionInitializer.INITIAL_MEAN_BIAS;
+        double biasVariance = AlleleFractionInitializer.INITIAL_BIAS_VARIANCE;
+        double previousIterationLogLikelihood;
+        double nextIterationLogLikelihood = Double.NEGATIVE_INFINITY;
+        logger.info(String.format("Initializing MLE hyperparameter values for allelic panel of normals.  Iterating until log likelihood converges to within %.3f.",
+                AlleleFractionInitializer.LOG_LIKELIHOOD_CONVERGENCE_THRESHOLD));
+        int iteration = 1;
+        do {
+            previousIterationLogLikelihood = nextIterationLogLikelihood;
+            meanBias = estimateMeanBias(meanBias, biasVariance, counts);
+            biasVariance = estimateBiasVariance(meanBias, biasVariance, counts);
+            nextIterationLogLikelihood = AlleleFractionLikelihoods.logLikelihoodForAllelicPanelOfNormals(meanBias, biasVariance, counts);
+            logger.info(String.format("Iteration %d, model log likelihood = %.3f.", iteration, nextIterationLogLikelihood));
+            iteration++;
+        } while (iteration < AlleleFractionInitializer.MAX_ITERATIONS &&
```

Comment:
Should the method give some kind of warning if it didn't converge during first `AlleleFractionInitializer.MAX_ITERATIONS` iterations?